pkg/volume/gce_pd/gce_pd.go

```diff
@@ -84,12 +86,7 @@ func (plugin *gcePersistentDiskPlugin) GetPluginName() string {
 }
 
 func (plugin *gcePersistentDiskPlugin) GetVolumeName(spec *volume.Spec) (string, error) {
```

Comment:
Do we still need this function? Maybe it should have old behavior and be called `GetPDName` or if it just has the new behavior, why can't we just use `GetDeviceName` directly.